Godeps/_workspace/src/k8s.io/kubernetes/cmd/genutils/genutils.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package genutils
```

Comment:
OK.

On Wed, Jun 22, 2016 at 2:16 PM, Juan Vallejo <notifications@github.com>
wrote:

> In Godeps/_workspace/src/k8s.io/kubernetes/cmd/genutils/genutils.go
> <https://github.com/openshift/origin/pull/9448#discussion_r68106188>:
>
> > +Copyright 2015 The Kubernetes Authors All rights reserved.
> > +
> > +Licensed under the Apache License, Version 2.0 (the "License");
> > +you may not use this file except in compliance with the License.
> > +You may obtain a copy of the License at
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package genutils
>
> @smarterclayton <https://github.com/smarterclayton> It's present in newer
> versions of k8s:
> https://github.com/kubernetes/kubernetes/tree/master/cmd/genutils
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9448/files/1c1c7e7d56c397f95ad0f520bee03751ee160c72#r68106188>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p1r2WHZWvEY7DJLYUhl46nQLStLIks5qOXwLgaJpZM4I6J0X>
> .
>